deploy/helm/kubecf/values.yaml

```
@@ -238,6 +238,8 @@ features:
     registry:
       service:
         nodePort: 32123
+    cluster_role:
+          rules: []
```

It's missing the keys from before.

```suggestion
    rbac:
      webhook:
        cluster_role:
          rules: []
```